due-processed/template-parts/icon-twitter.php

```diff
@@ -0,0 +1,3 @@
+<svg class="icon" width="18" height="15" viewBox="0 0 18 15" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
remove extra attributes
```suggestion
<svg class="icon" viewBox="0 0 18 15">
```